src/main/resources/assets/tablesaw/models/block/tablesaw.json

```diff
@@ -72,14 +72,22 @@
 				"west": {"uv": [1, 8, 15, 16], "texture": "#2"}
 			}
 		},
+		{
+			"from": [8, 6, 1],
+			"to": [8, 14, 15],
+			"faces": {
+				"east": {"uv": [15, 16, 1, 8], "texture": "#2"},
+				"west": {"uv": [15, 16, 1, 8], "texture": "#2"}
+			}
+		},
 		{
 			"from": [0, 15, 0],
-			"to": [16, 15.5, 16],
+			"to": [16, 16, 16],
```

Comment:
This is un-mixeling the top plate? If so can we update the icon and make sure the outlineShape agrees in the block?

Reply:
The block outline extended past the "mixeled" guard plate as shown here: 
![image](https://user-images.githubusercontent.com/17211100/202870691-4e697a71-f9c5-4141-b3c8-99b36d0ab1d0.png)

This is what it looks like now: 
![image](https://user-images.githubusercontent.com/17211100/202871474-84e44f43-a6d2-4f4c-939c-acff37791f19.png)